src/test/java/edu/hm/hafner/analysis/IssueAssert.java

```diff
@@ -0,0 +1,139 @@
+package edu.hm.hafner.analysis;
+
+import java.util.Objects;
+
+import org.assertj.core.api.AbstractAssert;
+
+/**
+ * Custom assertions for {@link Issue}.
+ */
+public class IssueAssert extends AbstractAssert<IssueAssert, Issue> {
+
+    public IssueAssert(final Issue actual) {
+        super(actual, IssueAssert.class);
+    }
+
+    public static IssueAssert assertThat(Issue actual) {
+        return new IssueAssert(actual);
+    }
+
+    public IssueAssert hasFileName(String fileName) {
+        isNotNull();
+
+        if (!Objects.equals(actual.getFileName(), fileName)) {
+            failWithMessage("Expected fileName to be <%s> but was <%s>", fileName, actual.getFileName());
```

Comment:
Der Standard ist:
`private static final String EXPECTED_BUT_WAS_MESSAGE = "%nExpecting:%n <%s>%nto be equal to:%n <%s>%nbut was not.";`

Reply:
Done.

Ist dies ein Standard im Jenkins-Projekt? 

Ich habe mich am Beispiel in der AssertJ-Dokumentation orientiert: http://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/AbstractAssert.html#failWithMessage(java.lang.String,%20java.lang.Object...)